crates/rspack_core/src/old_cache/local/code_splitting_cache.rs

```diff
@@ -38,7 +39,12 @@ where
     return Ok(());
   }
 
-  if !compilation.has_module_import_export_change() {
+  let has_change = compilation.has_module_import_export_change();
```

Comment:
IMO we can remove this once incremental build chunk graph get stable enough, thoughts?

Reply:
Agree